src/main/scala/cromwell/binding/types/WdlType.scala

```diff
@@ -29,3 +29,14 @@ trait WdlType {
 
   def toWdlString: String
 }
+
+object WdlType {
+
+  private lazy val wdlTypes = Seq(
```

Comment:
If this can be a java `enum` we'd get this collection for free with `values`, but I have doubts it can...